llvm/lib/Target/ARM/ARMAsmPrinter.cpp

```diff
@@ -171,7 +171,9 @@ bool ARMAsmPrinter::runOnMachineFunction(MachineFunction &MF) {
   // These are created per function, rather than per TU, since it's
   // relatively easy to exceed the thumb branch range within a TU.
   if (! ThumbIndirectPads.empty()) {
-    OutStreamer->emitAssemblerFlag(MCAF_Code16);
+    auto &TS =
```

Comment:
ARMAsmPrinter should probably define a convenience method that returns ARMTargetStreamer. The cast appears to occur in quite a few functions.